src/compile/scale.ts

```diff
@@ -65,31 +65,28 @@ export function type(fieldDef: FieldDef, channel: Channel, mark: Mark): string {
         // Also, if we support color ramp, this should be ordinal too.
         return 'time'; // time has order, so use interpolated ordinal color scale.
       }
-      if (contains([ROW, COLUMN, SHAPE], channel)) {
```

Comment:
Please undo this.  This will not output any thing reasonable.  
When would you want a value of shape to be linear value except than you're making mistake? 
When would you map a facet to a quantitative scale?